python.wake

```diff
@@ -29,15 +29,16 @@
 ##########################################################
 global def addPythonEnv pipDir plan =
 
-    # Install the virtual environment if not already done
-    def installed = installPipenvEnv pipDir
-    def venv = "build/{pipDir}/.venv"
+  # Install the virtual environment if not already done
+  def installed = installPipenvEnv pipDir
+  def venv = "build/{pipDir}/.venv"
 
-    # Add the binary directory to PATH.
-    plan
-    | editPlanVisible (installed ++ _)
-    | addPlanEnvironmentPath  "PYTHONPATH"  venv
-    | addPlanEnvironmentPath  "PATH" "{venv}/bin"
+  # Add the binary directory to PATH.
+  plan
+  | editPlanVisible (installed ++ _)
+  | addPlanEnvironmentPath  "PYTHONPATH"  venv
+  | addPlanEnvironmentPath  "PYTHONPATH"  "{venv}/lib/python3.7"
+  | addPlanEnvironmentPath  "PATH" "{venv}/bin"
```

Comment:
these are relative paths right? what if the plan is running in a different directory?

Reply:
We discussed this offline and acknowledged that yes, these paths will break if we run the plan from a different directory. However, we have other problems we need to solve in order for these to be more relocatable, such as turning this into a proper Runner and doing more of this installation as a wit plugin rather than as a wake rule.

Upon reading this block of code more carefully, I think the one, non-whitespace change that I made is actually not necessary because I figured out the virtualenv `--copies` flag, so let me at least revert that part.